package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "workspace-client",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "typings/index.d.ts",
+  "scripts": {
+    "build": "rimraf dist && webpack-cli --mode production --config webpack.config.js",
+    "test": "jest",
+    "prepare": "npm run test && npm run build"
+  },
+  "contributors": [
+    {
+      "name": "Oleksii Kurinnyi",
+      "email": "okurinny@redhat.com"
+    }
+  ],
+  "license": "EPL-2",
+  "dependencies": {
+    "axios": "~0.18.0",
+    "moxios": "git://github.com/stoplightio/moxios#v1.3.0",
+    "websocket": "~1.0.23"
+  },
+  "devDependencies": {
+    "@types/jest": "^22.1.3",
```

Comment:
we should stick for dev dependencies as well

Reply:
done